### PR TITLE
TAU: Respect ~fortran for +mpi

### PR DESCRIPTION
**Pull request by wspear**

### CODE REVIEW

**Comment by naromero77** [comment]
Thanks, this is the exact patch I used in my local sandbox. IIRC correctly, one of the maintainers to approve, before I can merge in the patch.

**Review by khuck — APPROVED** [review]
LGTM

**Review by eugeneswalker — APPROVED** [review]

**Pull request merged**
